src/services/twilio-help/twilio-command-help.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const CommandHelp = require('@oclif/plugin-help/lib/command.js');
+const list = require('@oclif/plugin-help/lib/list');
+const chalk = require('chalk');
+const indent = require('indent-string');
+
+/**
+ * Extended functionality from @oclif/plugin-help.
+ * Link: https://github.com/oclif/plugin-help
+ * author: onuzbee
+ */
+class TwilioCommandHelp extends CommandHelp.default {
+  // Override parent functionality
+  flags(flags) {
+    if (flags.length === 0) return '';
+
+    const optionalFlags = flags.filter((f) => !f.required);
+    const optionalBody = this.generateFlagsOutput(optionalFlags);
+    const requiredFlags = flags.filter((f) => f.required);
+    const requiredBody = this.generateFlagsOutput(requiredFlags);
+
+    const returnList = [chalk.bold('OPTIONS')];
+
+    if (requiredFlags.length > 0) {
+      returnList.push(chalk.bold('REQUIRED FLAGS'));
+      returnList.push(indent(requiredBody, 2));
+    }
+
+    returnList.push(chalk.bold('OPTIONAL FLAGS'));
+    returnList.push(indent(optionalBody, 2));
+    return returnList.join('\n');
+  }
+
+  /**
+   *   Forked and refactored from oclif default implementation.
+   *   Link: https://github.com/oclif/plugin-help/blob/master/src/command.ts#L125-L154
+   */
+  generateFlagsOutput(flags) {
```

Comment:
Can you break this function into smaller functions? There is a sonar code smell as the cognitive complexity is over 15: https://sonarcloud.io/project/issues?id=twilio_twilio-cli&pullRequest=281&resolved=false

Reply:
This is a forked version from the parent implementation, and haven't modified it (And has UTs for indentation in the parent repo). 
Modifying this piece to a reduced cognitive complexity, would also require us to imitate back all those tests in our repo. Please let me know if this seems critical, and we'd need to pick up as a separate refactoring.

Reply:
We can ignore it then. But I'm not really sure about what the guidelines are w.r.t sonar bugs